### DISCUSSION TITLE
DeviceConnected message possible issue

### DISCUSSION THREAD

**Discussion by bakhshetyan** [Koenkk/zigbee2mqtt#5465] [General]
I supose there is an issue with zigbee2mqtt/bridge/log ... "type":"device_connected" mqtt message:

`{"message":{"friendly_name":"0x00158d000241b5df"},
"type":"device_connected"}`

In all other messages of the topic **message** property contains string identifier and **meta** property provides a list of friendly_name etc ...

Is it intended or just a typo ?

**Comment by Koenkk** [comment]
This message comes from the legacy api wich will be deprecated with the next release. (#3281). Therefore no fixes/changes will be done to this.
